hw1/02-palindrome.js

```diff
@@ -1,3 +1,46 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", checkPalindrome);
+
+function checkPalindrome() {
+  const value = document.querySelector("input").value;
```

Comment:
Since the `input` element is not being added/removed, `querySelector` will always return the same DOM element. You should be able to reuse the `elem` variable that is defined above instead of calling `document.querySelector("input")` again here.

Reply:
Thanks for noticing this! I have updated it now